lib/Github/HttpClient/Plugin/GithubExceptionThrower.php

```diff
@@ -59,7 +59,12 @@ public function doHandleRequest(RequestInterface $request, callable $next, calla
                 if (422 === $response->getStatusCode() && isset($content['errors'])) {
                     $errors = [];
                     foreach ($content['errors'] as $error) {
-                        switch ($error['code']) {
+                        $code = null;
```

Comment:
when no code was contained, the logic in the default-case of the switch statement worked for me.

this PR is just about removing the php warning which was emitted.